official/projects/detr/configs/detr.py

```diff
@@ -101,3 +157,140 @@ def detr_coco() -> cfg.ExperimentConfig:
           'task.train_data.is_training != None',
       ])
   return config
+
+COCO_INPUT_PATH_BASE = ''
+COCO_TRAIN_EXAMPLES = 118287
+COCO_VAL_EXAMPLES = 5000
+
+@exp_factory.register_config_factory('detr_coco_tfrecord')
+def detr_coco() -> cfg.ExperimentConfig:
+  """Config to get results that matches the paper."""
+  train_batch_size = 64
+  eval_batch_size = 64
+  steps_per_epoch = COCO_TRAIN_EXAMPLES // train_batch_size
+  train_steps = 300 * steps_per_epoch  # 300 epochs
+  decay_at = train_steps - 100 * steps_per_epoch  # 200 epochs
+  config = cfg.ExperimentConfig(
+      task=DetrTask(
+          init_checkpoint='gs://tf_model_garden/vision/resnet50_imagenet/ckpt-62400',
+          init_checkpoint_modules='backbone',
+          annotation_file=os.path.join(COCO_INPUT_PATH_BASE,
+                                       'instances_val2017.json'),
+          model=Detr(
+              input_size=[1333, 1333, 3],
+              norm_activation=common.NormActivation(use_sync_bn=False)),
```

Comment:
shhould use_sync_bn default to True?

Reply:
I don't think so. I will change it.

Reply:
The default is defined in the "official.vision.configs import common".
I think it is out of my hands.

Reply:
No, I mean should this line be
norm_activation=common.NormActivation(use_sync_bn=True)) ?
I think in DETR's original setup it is set to True.

Reply:
nvm, I think we freeze the backbone.
https://github.com/tensorflow/models/blob/d81c7d4b35404bae2a7f3583668fdd1cfc51fce5/official/projects/detr/modeling/detr.py#L119

Reply:
Thank you for the review. I changed the use_sync_bn to True. :)

Reply:
We might need to add a field ```bn_trainable```  here?
https://github.com/tensorflow/models/blob/d81c7d4b35404bae2a7f3583668fdd1cfc51fce5/official/vision/configs/common.py#L113

Reply:
Can I modify the common file? If I can, I will add the argument.

Reply:
Never mind. I read the code more carefully, you don't need to make any changes, setting sync_batch_norm to True looks good as it shouldn't matter.

https://github.com/tensorflow/models/blob/d81c7d4b35404bae2a7f3583668fdd1cfc51fce5/official/vision/modeling/backbones/resnet.py#L432